### DISCUSSION TITLE
Running ppx_inline_test in parallel

### DISCUSSION THREAD

**Discussion by naim-mr** [ocaml/dune#9175] [Q&A] [answered]
Hi, 
I am trying to run ppx_inline tests in parallel and I have some trouble. But I dont get where I have to specified the "inline-test-lib lib:partition" argument.

I have three tests files "test1.ml" "test2.ml" "test3.ml" and here is my dune file which doesn't work.

```(library
   (name tests)
   (inline_tests (flags (-partition:test1))
  (preprocess
   (pps ppx_inline_test -inline-test-lib lib:test1)))
  ```
  And I get the following error: 
```Error: Library "lib:test1" not found.```  which I understand but I just did not find how to specify my dune file to correct it. 

So I am asking the community for help please :)

**Comment by hhugo** [comment, marked as answer]
Running ppx_inline_test in parallel should work automatically once you use `ppx_inline_test.v0.16.1`.
You can't really control the behavior from a library using ppx_inline_test.

**Reply by naim-mr** [reply]
Ok, perfect thanks!
